multi_map_it_test.go

```diff
@@ -539,6 +539,25 @@ func TestMultiMap_ContainsEntry(t *testing.T) {
 	})
 }
 
+func TestMultiMap_ValueCount(t *testing.T) {
+	it.MultiMapTester(t, func(t *testing.T, m *hz.MultiMap) {
+		ctx := context.Background()
+		key := "key"
+		targetValues := []interface{}{"v1", "v2", "v3"}
+		it.Must(m.PutAll(ctx, key, targetValues...))
+		count, err := m.ValueCount(ctx, key)
+		if err != nil {
+			t.Fatal(err)
+		}
+		assert.EqualValues(t, count, len(targetValues))
```

Comment:
Same thing @emreyigit mentions is also present here

Reply:
@nevzatseferoglu I think @utku-caglayan 's comment is not yet resolved. Could you resolve it and re-request reviews?